app/views/categories/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="trade">
+  <div class="header">
+    <p><%= @category.name %></p>
+  <p>$<%= @total %></p>
+  </div>
+  <% if @total == 0 %>
+  <%= link_to "Add new trade", new_category_trade_path(@category), class: "bg-blue-400 text-white py-2 px-12 text-lg rounded-lg mt-6 absolute bottom-10" %> |
+  <%= link_to "Back", categories_path, class: "text-blue-500 py-2 px-4 rounded-lg absolute bottom-2" %>
+  <p>No trades found</p>
+  <img src="https://cdn-icons-png.flaticon.com/512/8541/8541407.png" alt="No trades found">
+<% else %>
+    <div class="cards">
+      <% @trades.each do |trade| %>
+      <div class="card">
+        <%= image_tag(@category.icon, alt: "Icon pic", class: "left") %>
+        <div class="right">
+          <div class="sub-right">
+            <p class="trade-name"><%= trade.name %></p>
+            <p class="trade-amount">$<%= trade.amount %></p>
```

Comment:
- [ ] Whats expected is the amount of transaction in a category, not the price. Lets assume you have 4 transation, 6 should be there not the prices of the 6.

### Instructions:
![image](https://github.com/MenendezON/Budget-app/assets/50990925/137d2a7d-b8e7-4e14-93fb-33de0c911e87)

### Your view:
![image](https://github.com/MenendezON/Budget-app/assets/50990925/ddf9830b-0948-4f27-8130-abe09522c059)